src/backend/core/models.py

```diff
@@ -288,7 +288,8 @@ class Meta:
 
     def __str__(self):
         main_str = "[main]" if self.is_main else ""
-        return f"{self.email:s}{main_str:s}"
+        id_str = self.email or self.sub
```

Comment:
I would have expected it to fix the display in the admin, but I still have "-" when no email are given 🤔